Resources/Locale/en-US/ui/launcher.ftl

```diff
@@ -0,0 +1,4 @@
+### UI
+
+launcher-connecting = Failed to connect to server:
```

Comment:
The name of this localization string makes no sense since it's specifically for connection error.